Azure/sonic-buildimage
# [202405][Nokia-7220 IXR] Add support for NOKIA 7220 IXR-H4

#### Why I did it
Add support for NOKIA 7220 IXR-H4

> NOKIA 7220-IXR-H4:
> Platform: x86_64-nokia_ixr7220_h4-r0
> HwSKU: Nokia-IXR7220-H4
> ASIC: Broadcom
> Port Config: 64x400G + 2x10G

Add device files for NOKIA 7220 IXR-H4-32D

##### Work item tracking
- Microsoft ADO **(number only)**:

#### How I did it
Added module under ../device/nokia directory.
Modified the platform/broadcom/one-image.mk and platform/broadcom/ platform-modules-nokia.mk

#### How to verify it

1. Make sure the sonic-buildimage is successful

2. Run this image on x86_64-nokia_ixr7220_h4-r0 and x86_64-nokia_ixr7220_h4_32d-r0 and verify all dockers are up and test basic commands like:

- show version
- show platform summary
- show platform syseeprom
- show platform fan
- show platform psustatus
- show platform firmware status
- show platform temperature
- sudo show system-health detail
- show interface status

3. Run OC test, for T0/T1 topology and have 100% pass on platform.

#### Which release branch to backport (provide reason below if selected)




#### Tested branch (Please provide the tested image version)




#### Description for the changelog




#### Link to config_db schema for YANG module changes


#### A picture of a cute animal (not mandatory but encouraged)
